tardis/montecarlo/montecarlo.pyx

```diff
@@ -74,7 +79,8 @@ cdef extern from "src/cmontecarlo.h":
         double inverse_sigma_thomson
         double inner_boundary_albedo
         int_type_t reflective_inner_boundary
-        double *chi_bf_tmp_partial
+        photo_xsect_1level ** photo_xsect
```

Comment:
I'm not familiar with the exact usecase but I think double pointers can easily and should be avoided here.